server/routes/handlers/projects.js

```diff
@@ -146,6 +146,20 @@ module.exports = {
             { label: 'Supervisor Phone', value: 'supervisor.phone' },
           ];
 
+          const answers = data.map(x => x.answers || {});
+          const answerKeys = [
+            ...new Set((answers || []).reduce((acc, val) => {
```

Comment:
You could default the value of answers when it's actually defined rather than patching it on usage

Reply:
Or just load the event and grab the array of answer keys from there.

Reply:
which should available in req.app.locals.event